media/main.js

```diff
@@ -22,6 +22,17 @@ function () {
 	//connect to the vscode api
 	const vscode = acquireVsCodeApi();
 
+	class Window_History {  
+		Back() {
+			window.history.back();
+		}
+		Forward() {
+			window.history.forward();
+		}
+	}
+
+	var obj = new Window_History();
+
 	document.body.addEventListener('click', event => {
 		let node = event && event.target;
```

Comment:
node is not changed any more so could be made a const with a type.

Reply:
const maybe, but see the notes about JS vs. TS in this case, so no type

Reply:
my fault